runner/product/rcptt.runner.product

```diff
@@ -276,7 +279,7 @@
       <plugin id="org.eclipse.rcptt.watson.jobs"/>
       <plugin id="org.eclipse.rcptt.zephyr"/>
       <plugin id="org.eclipse.swt"/>
-      <plugin id="org.eclipse.swt.cocoa.macosx.x86_64"/>
+      <plugin id="org.eclipse.swt.cocoa.macosx.x86_64" fragment="true"/>
       <plugin id="org.eclipse.swt.gtk.linux.x86_64"/>
```

Comment:
This should also be `fragment="true"/>`?

Reply:
Fragment attribute was added by PDE automatically. I'm not aware of its significance. Runner on Linux still works, so at least Tycho was not confused by this. This will eventually become consistent once we contribute from more platforms. I'm content and lazy, so I'm going to leave it as is. Some education about this new PDE quirk would not hurt though.